source/postprocess/visualization.cc

```diff
@@ -129,7 +129,7 @@ namespace aspect
 
       /**
        * This Postprocessor will generate the output variables of mesh velocity
-       * for when a deforming mesh is used.
+       * for when a deforming mesh is used. 
```

Comment:
Some script or other is probably going to complain about the single space you added at the end of the line here.